pkgs/native_assets_builder/test_data/treeshaking_native_libs/test/treeshaking_native_libs_test.dart

```diff
@@ -0,0 +1,13 @@
+// Copyright (c) 2023, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'package:test/test.dart';
+import 'package:treeshaking_native_libs/treeshaking_native_libs.dart';
+
+void main() {
```

Comment:
I love this test/example!

Please exercise this test_data somewhere in pkgs/native_assets_builder/test/**_test.dart so that we're sure it works and keeps working :)